cvat-ui/src/reducers/interfaces.ts

```diff
@@ -505,6 +505,11 @@ export interface AnnotationState {
         visible: boolean;
         data: any;
     };
+    removeinrange: {
+        sessionInstance: any | null;
+        startFrame: number;
+        endFrame: number;
+    };
```

Comment:
I do not think that the state should be global. It is only used in one Modal window.